spec/std/float_printer_spec.cr

```diff
@@ -107,8 +107,8 @@ describe "#print Float64" do
     test_pair 5.5626846462680035e-309, "5.562684646268003e-309"
   end
 
-  pending_win32 "failure case" do
-    # grisu cannot do this number, so it should fall back to libc
+  it "grisu failure cases" do
+    test_pair 4.91e-6, "4.91e-6"
     test_pair 3.5844466002796428e+298, "3.5844466002796428e+298"
   end
```

Comment:
Since Dragonbox does not depend on `LibC`, this ought to pass on Windows too. (If our `sprintf` also depends on Dragonbox, then those strings in turn would become platform-independent.)